logicrunner/builtin/contract/deposit/deposit.go

```diff
@@ -26,16 +26,17 @@ import (
 	"github.com/insolar/insolar/logicrunner/goplugin/foundation"
 )
 
-type DepositStatus string
+type status string
 
 const (
-	DepositConfirms uint = 3
+	confirms uint = 3
```

Comment:
Make `confirms` first const statement and all `status*` second const statement, cause they have different logic of use. (see example in `insolar/bus/bus.go`)